src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -202,7 +192,7 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
                             store={store}
                             key={store.hugoGeneSymbols.join(",")}
                             routing={this.props.routing}
-                            isVirtualStudy={getBrowserWindow().currentQueryStore.isVirtualStudyQuery}
+                            isVirtualStudy={this.resultsViewPageStore.virtualStudies.result!.length > 0} // will be complete b/c store.studies is dependent on it
```

Comment:
if this can be derived from the store, should we move that into ResultsViewOncoprint, where we can also more cleanly explicitly do 
```
@computed get isVirtualStudy() {
    this.resultsViewPageStore.virtualStudies.isComplete && 
    this.resultsViewPageStore.virtualStudies.result!.length > 0;
}
```
or something like that

Reply:
👍